app/src/main/res/layout/view_donor_badge.xml

```diff
@@ -6,38 +6,36 @@
     android:layout_height="wrap_content"
     tools:parentTag="android.widget.FrameLayout">
 
-    <com.google.android.material.chip.Chip
+    <Button
         android:id="@+id/donorChip"
-        style="@style/Chip.Accessible.Icon"
+        style="@style/App.Button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:minHeight="0dp"
+        android:paddingTop="6dp"
+        android:paddingBottom="6dp"
         android:text="@string/contributions_dashboard_badge_donor"
-        android:textColor="?attr/paper_color"
-        android:visibility="gone"
-        app:chipBackgroundColor="?attr/progressive_color"
-        app:chipCornerRadius="16dp"
-        app:chipIcon="@drawable/ic_baseline_volunteer_activism_24"
-        app:chipIconTint="?attr/paper_color"
-        app:chipStrokeWidth="0dp"
-        app:rippleColor="?attr/overlay_color"
-        tools:text="Donor"
-        tools:visibility="gone" />
+        android:visibility="visible"
+        android:textSize="12sp"
+        app:icon="@drawable/ic_baseline_volunteer_activism_24"
+        app:iconTint="?attr/paper_color"
+        app:iconSize="16dp"
+        tools:visibility="visible" />
 
-    <com.google.android.material.chip.Chip
+    <Button
         android:id="@+id/becomeADonorChip"
-        style="@style/Chip.Accessible.Icon.Small"
+        style="@style/App.Button.Secondary"
```

Comment:
Can we apply the following style for a new theme, which it should have a border for this chip.

```
    <style name="App.Button.Outline" parent="@style/Widget.Material3.Button.OutlinedButton">
        <item name="android:minHeight">48dp</item>
        <item name="android:minWidth">0dp</item>
        <item name="android:paddingStart">16dp</item>
        <item name="android:paddingEnd">16dp</item>
        <item name="backgroundTint">?attr/paper_color</item>
        <item name="iconSize">24dp</item>
        <item name="iconTint">?attr/progressive_color</item>
        <item name="android:textColor">?attr/progressive_color</item>
        <item name="strokeColor">?attr/border_color</item>
    </style>
```